index.bs

```diff
@@ -4184,9 +4184,9 @@ Note: In case of an arrow function in <code>functionDeclaration</code>, the
 
       ScriptCallFunctionParameters = {
         functionDeclaration: text;
+        awaitPromise: bool;
         ?arguments: [ArgumentValue];
         ?this: ArgumentValue;
-        ?awaitPromise: bool;
         target: Target;
```

Comment:
If the reason to move it up is that we have all required arguments first, we probably should do the same with `target` as well. Maybe it could be done now given that you modify the surrounding code in your PR.

Reply:
Moved all the optional params to the end.